src/App.tsx

```diff
@@ -1,22 +1,43 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+
+  const getMovies = (movies: Movie[], userQuery: string) => {
```

Comment:
`query` is already in scope
```suggestion
  const getMovies = (movies: Movie[]) => {
```